### ISSUE TITLE
`DrawerContent` `MenuItem`s not visible with larger font sizes or zoomed viewports [accessibility, a11y]

### ISSUE THREAD

**Issue by ashleemboyer** [bluesky-social/social-app#1297]
**Describe the bug**

When either system/browser text size or viewport zoom is increased, the `MenuItem`s rendered in `DrawerContent` (Search, Home, Notifications, etc.) are no longer visible. This renders the items useless for [an unpredictable number of users](https://viewports.fyi/) across device types and sizes. This is an accessibility bug related to [WCAG 2.1 Success Criterion 1.4.10 - Reflow (Level AA)](https://www.w3.org/TR/WCAG21/#reflow).

**To Reproduce**

Steps to reproduce the behavior _just one way_, and _in web_:

1. Go to https://bsky.app/
2. Zoom the interface to 200% (e.g. using the `cmd` and `+` keys)
3. Open the lefthand drawer using the hamburger icon at the top left of the viewport
4. Observe that the Search, Home, Notifications, etc. `MenuItem`s are not visible

**Expected behavior**

1. Go to https://bsky.app/
2. Zoom the interface to 200% (e.g. using the `cmd` and `+` keys)
3. Open the lefthand drawer using the hamburger icon at the top left of the viewport
4. The Search, Home, Notifications, etc. `MenuItem`s should be visible

**Screenshots**

<img width="1440" alt="Web view after following reproduction steps listed above" src="https://github.com/bluesky-social/social-app/assets/43934258/eb96d78a-9118-463b-a3cf-040d40715370">

**Details**

 - Platform: all, but these test steps were created from web
 - Platform version: 1.47.0

**Additional context**

Potential solution, which I have tested locally in web (but couldn't get my environment working in order to test mobile):

First, replace the `View` with `ScrollView` in `ShellInner` (`src/view/shell/index.web.tsx`).

<img width="1440" alt="VS Code diff view of the first described change" src="https://github.com/bluesky-social/social-app/assets/43934258/7f7dda11-8745-48a7-9a64-2fdde7dc23dc">

Second, replace the `ScrollView` with `View` in `DrawerContent` (`src/view/shell/Drawer.tsx`).

<img width="1440" alt="VS Code diff view of the second described change" src="https://github.com/bluesky-social/social-app/assets/43934258/63069ebd-1cd3-4033-8ee5-33ad4de0464c">

These changes make the whole `DrawerContent` scrollable rather than having it squish significantly and providing a tiny scrollable area for the `MenuItem`s.

**Comment by pfrazee** [comment]
Thanks for reporting! On it

**Comment by estrattonbailey** [comment]
Hey @ashleemboyer! I believe this issue is fixed in our latest release, v1.51. Thanks again for reporting!
